styles/index.scss

```diff
@@ -57,6 +58,10 @@ section {
   @container body (min-width: 768px) {
     padding: 200px 72px 0;
     max-width: 1024px;
+
+    &.max-width-686 {
```

Comment:
Good call on adding these _.max-width-686_ and _.-center_ classes--those will definitely be useful for other parts of the site!